scripts/ci/libraries/_initialization.sh

```diff
@@ -59,8 +59,6 @@ function initialize_common_environment {
     # Create useful directories if not yet created
     mkdir -p "${AIRFLOW_SOURCES}/.mypy_cache"
     mkdir -p "${AIRFLOW_SOURCES}/logs"
-    mkdir -p "${AIRFLOW_SOURCES}/tmp"
```

Comment:
Maybe better to remove the two lines above ? I  think it's nice to have the mkdirs in one "group" :)

Reply:
But the two lines above do use the variables which will be exported. So I think it makes more sense to `mkdir` those directly. Otherwise there could be discrepancies between `TMP_DIR` and `${AIRFLOW_SOURCES}/tmp` for example.